llama_stack/apis/vector_dbs/vector_dbs.py

```diff
@@ -34,6 +34,7 @@ class VectorDBInput(BaseModel):
     vector_db_id: str
     embedding_model: str
     embedding_dimension: int
+    provider_id: str | None = None
```

Comment:
Shouldn't provider_id be mandatory? Otherwise, how do we determine to which provider we should be routing to?

Reply:
Would you kindly raise an issue with the future work ?

Reply:
i see u raised https://github.com/meta-llama/llama-stack/issues/2834 

i have signed up for it 👍🏽